### PR TITLE
fixed aspect ratio display issue of photo

### PR DESCRIPTION
**Pull request by sarab97**
Fix the photo getting croped with unusual aspect ratios

**Changes**
minor css fix

**Issues**
#857 
#1116

### CODE REVIEW

File: src/components/slideshow/style.css [hunk 1]
```diff
@@ -141,4 +141,6 @@
 
 .slider-zoom-container {
     margin: auto;
+    max-height: 100% !important;
+    max-width: 100% !important;
```
Comment (dmitrylyzo) [inline src/components/slideshow/style.css:145]: I tried without `!important` - seems to work too.

Comment (sarab97) [reply src/components/slideshow/style.css:145]: problem seems to be fixed for firefox now. Can you elaborate on safari issue

**Comment by sarab97** [comment]
> Chrome 79: OK
> Firefox 72,75: image is not in the center.
> 
> Also tried on Safari 12 (iPhone5S): doesn't respect orientation - I am not sure why.

problem seems to be fixed for firefox now. Can you elaborate on safari issue

**Comment by dmitrylyzo** [comment]
Chrome 79, Firefox 72/75 - OK

Safari 12 (iPhone5S) / 13 (iPad Pro) still doesn't respect orientation. Tried Desktop and Mobile layouts.
I tried to revert zooming PR - Safari displays image fitted.

**Comment by dmitrylyzo** [comment]
iPhone5S
<details>
<summary>Portrait</summary>

_Suspicious that the top panel is here? Manifest seems correct_

IMG_3532
IMG_3533
</details>
<details>
<summary>Landscape</summary>

IMG_3531
IMG_3530
</details>

I can say that it respects width, because "landscaped" images are fitted.

**Pull request merged**